tests/propertyBinding/CMakeLists.txt

```diff
@@ -4,9 +4,18 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_AUTOMOC ON)
 
 facelift_add_qml_plugin(PropertyBindingCpp
-   URI tests.propertybinding
-   OUTPUT_BASE_DIRECTORY imports/cpp
-   SOURCES plugin/PropertyBindingPlugin.cpp
-   LINK_LIBRARIES PropertyBindingInterface
-)
+    URI tests.propertybinding
+    OUTPUT_BASE_DIRECTORY imports/cpp
+    SOURCES plugin/PropertyBindingPlugin.cpp
+    impl/cpp/PropertyBindingInterfaceCppImplementation.h
+    LINK_LIBRARIES PropertyBindingInterface)
+
+facelift_add_qml_plugin(PropertyBindingQml
+    URI tests.propertybinding
+    OUTPUT_BASE_DIRECTORY imports/qml
+    SOURCES plugin/PropertyBindingPlugin.cpp
+    impl/cpp/PropertyBindingInterfaceCppImplementation.h
```

Comment:
align with previous file which also belongs to SOURCES
do the same for all in this file

Reply:
done